### DISCUSSION TITLE
How did I not know about this?!

### DISCUSSION THREAD

**Discussion by wrex** [testing-library/testing-playground#340] [General]
Just a bit of fan mail. 

I was struggling with a jest/testing-library matcher and eventually stumbled across a passing reference to `screen.logTestingPlaygroundURL()` while reading the testing-library docs.

My jaw hit the floor when I pasted in the url, started typing the matcher into the textbox below, and IMMEDIATELY saw a recommended matcher pop up that worked **perfectly** for what I was trying to do. I can't believe I didn't know about this!

I'm honestly shocked that I'd never even heard of this before. Brilliant! Thanks a million.

This deserves to be MUCH more prominent on the testing-library site.

**Comment by smeijer** [comment]
Thank you so much for the kind words. Seriously, it me as a lot. ❤️

Is there a specific location in the docs that you have in mind which would improve the visibility of testing playground? We all benefit from an easy as possible onboarding & developer experience. 

The [docs are open source](https://github.com/testing-library/testing-library-docs) as well. If you have the time, feel free to submit a change.

Thanks again.

**Reply by wrex** [reply]
No kidding, this thing has saved me a ton of work and is actually kind of fun.

I will submit a PR as soon as I finish a project I'm working on. I think the main thing is a screenshot or two of what the testing playground actually looks like and lets you see (easily!).

It's made writing tests for some form components infinitely easier. Seriously great.
